### ISSUE TITLE
Access problem to a project page

### ISSUE THREAD

**Issue by davidnicholls125** [johnvanbreda/naturespot-d8#192]
In addition to myself, there are two administrators set up to access the records page for the Rutland Natural History Project (link below) - but neither appear to be working. John Rodgers was added as an admin weeks ago but when I masquerade as his account I'm logged out. When I masquerade as the second admin entry, RNHS, it says I don't have access to this group. I added the RNHS admin earlier this afternoon so I'm not sure if there is a delay in it being recognised?
https://www.naturespot.org.uk/projects/explore?group_id=2420&implicit=f

**Comment by davidnicholls125** [comment]
I've made some progress - the John Rodgers account was an old duplicate so I've deleted that account and added his later account, username JSandJRodgers, as the additional project admi. However this doesn't work either - it gives the same message as the RNHS account, ie no access to the group....

**Comment by johnvanbreda** [comment]
Although I can see the problems here, I don't know what sequence of actions led to it being this way. The JSandJRodgers is actually an old account. Although the records in the database show this account was linked to the group, the link was flagged as deleted so not working. I've undeleted it and this account now works.
The RNHS account is a newer account and it looks like this account had never been linked to the warehouse properly. I edited it and saved it in Drupal to fix the link. This account was linked to the group and also now works.
